cmake/ConfigCPack.cmake

```diff
@@ -200,24 +198,45 @@ set(CPACK_RPM_PACKAGE_RELEASE
 string(REGEX REPLACE "([a-zA-Z])-([0-9])" "\\1\\2" CPACK_RPM_PACKAGE_RELEASE
                      "${CPACK_RPM_PACKAGE_RELEASE}")
 string(REPLACE "-" "~" CPACK_RPM_PACKAGE_RELEASE "${CPACK_RPM_PACKAGE_RELEASE}")
-if(DEFINED ENV{CPACK_RPM_PACKAGE_RELEASE})
-    set(CPACK_RPM_PACKAGE_RELEASE $ENV{CPACK_RPM_PACKAGE_RELEASE})
-endif()
 
-# Get rpm distro
-if(CPACK_RPM_PACKAGE_RELEASE)
-    set(CPACK_RPM_PACKAGE_RELEASE_DIST ON)
-endif()
 set(CPACK_RPM_FILE_NAME "RPM-DEFAULT")
+set(CPACK_RPM_PACKAGE_RELEASE_DIST ON)
 
 # -------------------------------------------------------------------------------------- #
 #
-# Prepare final version for the CPACK use
+# Prepare final CPACK parameters
 #
 # -------------------------------------------------------------------------------------- #
 
 set(CPACK_PACKAGE_VERSION
     "${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}"
     )
 
+if(DEFINED ENV{ROCM_LIBPATCH_VERSION})
+    set(CPACK_PACKAGE_VERSION "${CPACK_PACKAGE_VERSION}.$ENV{ROCM_LIBPATCH_VERSION}")
+else()
+    set(CPACK_PACKAGE_VERSION "${CPACK_PROJECT_VERSION}")
+endif()
+
+if(DEFINED ENV{CPACK_DEBIAN_PACKAGE_RELEASE})
+    set(CPACK_DEBIAN_PACKAGE_RELEASE $ENV{CPACK_DEBIAN_PACKAGE_RELEASE})
+endif()
+
+if(DEFINED ENV{CPACK_RPM_PACKAGE_RELEASE})
+    set(CPACK_RPM_PACKAGE_RELEASE $ENV{CPACK_RPM_PACKAGE_RELEASE})
+endif()
+
+message(STATUS "Packaging Information ... ")
+message(STATUS "  CPack package name: ${CPACK_PACKAGE_NAME}")
```

Comment:
I believe all of these should use the `omnitrace_add_feature(<VARIABLE> “<DESCRIPTION>")` function

Reply:
Using the output summary from the Omnitrace features, rather than the messages.